app/scripts/modules/core/src/presentation/forms/validation/ValidationMessage.tsx

```diff
@@ -43,7 +46,7 @@ export const ValidationMessage = (props: IValidationMessageProps) => {
   return (
     <div className={`ValidationMessage ${containerClassName || containerClassNames[type] || ''}`}>
       {showIcon && <i className={iconClassName || iconClassNames[type] || ''} />}
-      <div className="message">{message}</div>
+      <div className="message">{isString(message) ? <Markdown message={message} /> : { message }}</div>
```

Comment:
I think something about this syntax got me confused. can you help me figure out what `{ message }` is doing in this case? the way i'm reading it is that if `message` is anything other than a string, this will take the value of the `message` variable and make it the value of a `message` **field** inside an object, which will then get given to react to render. what am I missing?

Reply:
yes, nice catch!